<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="39" total="43" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Regression Suite" duration-ms="10979" started-at="2019-05-09T11:10:44Z" finished-at="2019-05-09T11:10:55Z">
    <groups>
      <group name="regression">
        <method signature="addingNewDetails.addEmployee()[pri:0, instance:gpmsUI.gpmsAutomation.addingNewDetails@5700d6b1]" name="addEmployee" class="gpmsUI.gpmsAutomation.addingNewDetails"/>
        <method signature="basicDetails.homePage()[pri:0, instance:gpmsUI.gpmsAutomation.basicDetails@1ce92674]" name="homePage" class="gpmsUI.gpmsAutomation.basicDetails"/>
        <method signature="basicDetails.loginTST3()[pri:0, instance:gpmsUI.gpmsAutomation.basicDetails@1ce92674]" name="loginTST3" class="gpmsUI.gpmsAutomation.basicDetails"/>
      </group> <!-- regression -->
    </groups>
    <test name="Test" duration-ms="10979" started-at="2019-05-09T11:10:44Z" finished-at="2019-05-09T11:10:55Z">
      <class name="gpmsUI.gpmsAutomation.basicDetails">
        <test-method status="PASS" signature="homePage()[pri:0, instance:gpmsUI.gpmsAutomation.basicDetails@1ce92674]" name="homePage" duration-ms="5278" started-at="2019-05-09T11:10:44Z" finished-at="2019-05-09T11:10:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homePage -->
        <test-method status="PASS" signature="loginTST3()[pri:0, instance:gpmsUI.gpmsAutomation.basicDetails@1ce92674]" name="loginTST3" duration-ms="1226" started-at="2019-05-09T11:10:49Z" finished-at="2019-05-09T11:10:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTST3 -->
      </class> <!-- gpmsUI.gpmsAutomation.basicDetails -->
      <class name="gpmsUI.gpmsAutomation.addingNewDetails">
        <test-method status="FAIL" signature="addEmployee()[pri:0, instance:gpmsUI.gpmsAutomation.addingNewDetails@5700d6b1]" name="addEmployee" duration-ms="1954" started-at="2019-05-09T11:10:51Z" finished-at="2019-05-09T11:10:53Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed: Add Employee step, as Employee Number'Test01' already exists]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed: Add Employee step, as Employee Number'Test01' already exists
at org.testng.Assert.fail(Assert.java:96)
at gpmsUI.gpmsAutomation.addingNewDetails.addEmployee(addingNewDetails.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addEmployee -->
        <test-method status="FAIL" signature="addNewCompany()[pri:0, instance:gpmsUI.gpmsAutomation.addingNewDetails@5700d6b1]" name="addNewCompany" duration-ms="2495" started-at="2019-05-09T11:10:53Z" finished-at="2019-05-09T11:10:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failed adding new Company Details]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failed adding new Company Details
at org.testng.Assert.fail(Assert.java:96)
at gpmsUI.gpmsAutomation.addingNewDetails.addNewCompany(addingNewDetails.java:100)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCompany -->
      </class> <!-- gpmsUI.gpmsAutomation.addingNewDetails -->
    </test> <!-- Test -->
  </suite> <!-- Regression Suite -->
</testng-results>
